databroker/replay/__init__.py

```diff
@@ -0,0 +1 @@
+from .replay import replay  # noqa F401
```

Comment:
```suggestion

```

Doing this makes the underlying module inaccessible due to the shadowing.  Can we accept the extra layer or rename `replay.py` -> `__init__.py` ?

Reply:
Somewhat on purpose (based on my interpretation of the design). Nothing else in that module is meant to be imported (private and main functions), and the main function can still be accessed by running the file or adding an entry point if needed. 

I have no qualms about changing it though if recommended style.